flexx/event/_component.py

```diff
@@ -219,6 +219,8 @@ def _comp_init_property_values(self, property_values):
         """ Initialize property values, combining given kwargs (in order)
         and default values.
         """
+        # Property values must be poped when consumed so that the remainer is used for 
+        # instantiation of the Widget
```

Comment:
Could you move this comment to the docstring above for extra visibility? And we can just say that they are popped so that the remainder can be used for something else. We're in a low-level part of Flexx here, and this code knows nothing of widgets :)

Reply:
Done